src/R/Support/Impl/Help/Packages/PackageIndex.cs

```diff
@@ -260,7 +262,7 @@ public sealed class PackageIndex : IPackageIndex {
             try {
                 await _host.StartSessionAsync();
                 return _host.LoadedPackageNames;
-            } catch (OperationCanceledException) { }
+            } catch (RException) { }
```

Comment:
You are right. But there is also a number of places when we call things on the session and yet not handle disconnected. Let me look there as well.